packages/mock/CMakeLists.txt

```diff
@@ -3,6 +3,7 @@ cmake_minimum_required(VERSION 3.5.0)
 project(soss-mock)
 
 find_package(soss-core REQUIRED)
+find_package(yaml-cpp REQUIRED)
```

Comment:
Similar to my previous comment, if we add `find_dependency(yaml-cpp)` to `soss-core`'s config-file, we shouldn't need this anymore.